android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt

```diff
@@ -168,7 +175,10 @@ internal class BetterPlayer(
             }
         } else if (clearKey != null && clearKey.isNotEmpty()) {
             drmSessionManager = if (Util.SDK_INT < 18) {
-                Log.e(TAG, "Protected content not supported on API levels below 18")
+                Log.e(
+                    TAG,
+                    "Protected content not supported on API levels below 18"
+                )
```

Comment:
Get rid of the line formatting changes please

Reply:
After rereading I think you might have meant for the entire PR, my bad. All the formatting I could clean up is cleaned up now.